samples/rest/bing_visual_search_v7.py

```diff
@@ -28,14 +29,18 @@ def visual_search_basic(
     Args:
         subscription_key (str): Azure subscription key of Bing Visual Search service
         auth_header_name (str): Name of the authorization header
+        image_url (str): Url of the image used for search
         image_path (str): Path of the image used for search
         mkt (str): Market to search in
     """
+    # Default value for demo purposes
```

Comment:
This looks unrelated to the PR. Can we split it in a follow up PR?

Reply:
If you're talking about the refactoring of `bing_visual_search_v7.py`, well, the tests were failing until I did the changes, so I figured they belong here.
Should I maybe open a PR and merge it (before?) this PR? the only disadvantage I could think of is the overhead. What are your thoughts?

Reply:
Yes please, just in case something needs to be reverted is not coupled